collections/ansible_collections/massopencloud/esi/roles/network/tasks/destroy_network.yaml

```diff
@@ -0,0 +1,7 @@
+- name: Check that exactly one network matching the ident exists  # noqa:no-changed-when
+  ansible.builtin.command: >-
+    openstack network show {{ network_identifier }} -f json
```

Comment:
is it required? We try to delete anyway, right?

Reply:
OpenStack allows multiple networks with the same name, so I figure we want to throw an error if someone tries to delete a network by name and it matches multiple networks.